### PR TITLE
Update Rich descriptions

### PR DESCRIPTION
**Pull request by LanaDX**
Update end user documentation for Rich Text Editor

### CODE REVIEW

File: interface-elements-for-web/articles/rich-text-editor/header-and-footer/header-and-footer.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Headers and Footers
+
+---
+# Headers and Footers
+
+Headers and footers are areas into the top and bottom margins of each page in a document. Headers and footers can hold text and graphics, e.g., a company logo, the document title or the author's name.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/rich-text-editor/header-and-footer/header-and-footer.md]: areas into?

Comment (dirk-pieterse) [reply interface-elements-for-web/articles/rich-text-editor/header-and-footer/header-and-footer.md]: can hold? Do you mean 'can display'?

File: interface-elements-for-web/articles/rich-text-editor/miscellaneous/insert-a-hyperlink.md [hunk 2]
```diff
@@ -1,19 +1,84 @@
...
+## Hyperlinks
+
+A hyperlink is a document range (for example, portion of text, image) that you can use to go to a webpage, to another place within the same document, or to create a new email message. 
+
+Click a hyperlink while pressing CTRL to navigate to the link.
+
+### Add a hyperlink
+* Select a document range and click **Hyperlink** in **Insert** tab's **Links** group, or right-click the selected document range and click **Hyperlink...** in the context menu to invoke the **Hyperlink** dialog box.
+ 
+*Use the option buttons at the top of the dialog to change the hyperlink's direction.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/rich-text-editor/miscellaneous/insert-a-hyperlink.md]: direction? Why direction?

File: interface-elements-for-web/articles/rich-text-editor/miscellaneous/insert-a-hyperlink.md [hunk 3]
```diff
@@ -1,19 +1,84 @@
...
+
+**Web page**
+
+![EUD_ASPxRichEdit_Insert_HyperlinkDialog-WebPage](../../../images/img118710.png)
+
+Inserts a hyperlink to a web page. 
+
+Specify the hyperlink navigation URL (required), text, and tooltip, and click **OK**.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/rich-text-editor/miscellaneous/insert-a-hyperlink.md]: 'hyperlink navigation URL' Do you mean hyperlink's navigation URL?

File: interface-elements-for-web/articles/rich-text-editor/table-of-contents.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+---
+title: Table of Contents
+
+---
+# Table of Contents
+
+Table of Contents allow navigate through a document's marked content (paragraphs, images, tables, and equations). Click a table's row while pressing CTRL to move to the heading or figure.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/rich-text-editor/table-of-contents.md]: allow navigate? Allow who or what to navigate?

File: interface-elements-for-web/articles/rich-text-editor/tables.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+## Context Tabs
+Use buttons in the **Design** and **Layout** context tabs to customize a table's design and layout settings. These tabs appear when a pointer is in a table element.
+
+![RichTextEditor-Tables-DesignTab](../../images/rich-text-editor-tables-design-tab.png)
+
+![RichTextEditor-Tables-LayoutTab](../../images/rich-text-editor-tables-layout-tab.png)
+
+
+## Context Menu
+Right-click a table element to display a context menu. The menu commands allows you to modify table properties and layout.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/rich-text-editor/tables.md]: modify table properties and layout -> modify the table's properties and layout.

Comment (dirk-pieterse) [reply interface-elements-for-web/articles/rich-text-editor/tables.md]: 'commands allows'?

File: interface-elements-for-web/articles/rich-text-editor/table-of-contents.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+## Remove an Image, Table, or Equation from a Table of Figures
+Clear a figure name and update the table to remove the figure from a table. 
+
+## Insert a Table of Contents or Figures
+Click the **Table of Contents** button in the **References** tab to insert a table of contexts. 
+
+Click the **Insert Table of Figures** button in the **References** tab and select a command to insert a table of figures.
+
+## Update a Table
+If a navigation table displays outdated information (outdated headings/captions or outdated page numbers) after changing the document, set the pointer in the table and click the **Update Table** button in the **References** ribbon tab to update the table and display the correct information.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/rich-text-editor/table-of-contents.md]: set the pointer?

File: interface-elements-for-web/articles/rich-text-editor/tables.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
+---
+title: Tables
+
+---
+# Tables
+You can create and edit tables in documents. 
+
+Click the **Table** button in **Insert** tab to insert a table. Then select the table size and click the corresponding square in the drop-down box.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/rich-text-editor/tables.md]: "click the corresponding square" What does 'corresponding' refer to?

File: interface-elements-for-web/articles/rich-text-editor/viewing-and-navigating/document-views.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
...
+
+---
+# Document Views
+Use ribbon buttons in **View** tab's **Document Views** group to switch document views.
+
+![EUD_ASPxRichEdit_ViewTab](../../../images/img117731.png)
+
+The **Simple View** displays a document in the entire editing area and ignores the page layout. In this mode, the editor does not split a document into pages and ignores headers, footers, and page layout settings: margins, orientation, size, columns, and breaks.
+
+The **Print Layout** displays a document such as it will be printed: split into pages, with headers, footers, and with applied page layout settings: margins, orientation, size, columns, and breaks.
```
Comment (dirk-pieterse) [inline interface-elements-for-web/articles/rich-text-editor/viewing-and-navigating/document-views.md]: "such as it will"?

**Review by dirk-pieterse — APPROVED** [review]

File: interface-elements-for-web/articles/rich-text-editor/miscellaneous/insert-a-hyperlink.md (hunk 2 above)
Comment (LanaDX) [reply interface-elements-for-web/articles/rich-text-editor/miscellaneous/insert-a-hyperlink.md]: changed to "hyperlink's destination"

File: interface-elements-for-web/articles/rich-text-editor/table-of-contents.md (hunk 4 above)
Comment (LanaDX) [reply interface-elements-for-web/articles/rich-text-editor/table-of-contents.md]: changed to 'The Table of Contents allows navigating through'

File: interface-elements-for-web/articles/rich-text-editor/tables.md (hunk 5 above)
Comment (LanaDX) [reply interface-elements-for-web/articles/rich-text-editor/tables.md]: Changed to 'You can use the menu's commands to modify the table's properties and layout.'

File: interface-elements-for-web/articles/rich-text-editor/table-of-contents.md (hunk 6 above)
Comment (LanaDX) [reply interface-elements-for-web/articles/rich-text-editor/table-of-contents.md]: changed to 'place the pointer'

File: interface-elements-for-web/articles/rich-text-editor/tables.md (hunk 7 above)
Comment (LanaDX) [reply interface-elements-for-web/articles/rich-text-editor/tables.md]: Changed to 'Click the **Table** button in the **Insert** tab and select the table size in the drop-down box.'

File: interface-elements-for-web/articles/rich-text-editor/viewing-and-navigating/document-views.md (hunk 8 above)
Comment (LanaDX) [reply interface-elements-for-web/articles/rich-text-editor/viewing-and-navigating/document-views.md]: changed to 'The **Print Layout** displays a document split into pages, with headers,...'

File: interface-elements-for-web/articles/rich-text-editor/miscellaneous/insert-a-hyperlink.md (hunk 3 above)
Comment (LanaDX) [reply interface-elements-for-web/articles/rich-text-editor/miscellaneous/insert-a-hyperlink.md]: changed

File: interface-elements-for-web/articles/rich-text-editor/table-of-contents.md (hunk 4 above)
Comment (dirk-pieterse) [reply interface-elements-for-web/articles/rich-text-editor/table-of-contents.md]: The Table of Contents allows you to navigate through

**Pull request merged**